<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Sae" failures="1" tests="1" name="testNgTestRunner.failed_testRunner" time="13.975" errors="0" timestamp="2022-07-15T10:28:13 IST" skipped="0">
  <testcase classname="testNgTestRunner.failed_testRunner" name="runScenario" time="13.975">
    <failure message="expected [No data] but found [Beetroot]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [No data] but found [Beetroot]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at stepDefinitions.dealsPageSteps.validating_both_for_product_match(dealsPageSteps.java:42)
at âœ½.validating both for product match(file:///C:/Users/gnana/OneDrive/Desktop/Cucumber/CucumberFrameWork1/src/test/java/features/searchProduct.feature:7)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- testNgTestRunner.failed_testRunner -->
